### ISSUE TITLE
Non images are being captured by the plugin

### ISSUE THREAD

**Issue by sinabahram** [coyote-team/coyote-wp#84]
We need to only submit images to Coyote, but right now all attachments e.g. PDF files, M4A files, mp4 files, are being submitted.

I suggest filtering on popular image extensions for now.

**Comment by jkva** [comment]
I'm unable to reproduce this. Implementation-wise there exists explicit filtering on anything not of mime-type `image/*`. 
When creating a new organisation and importing existing attachments and posts including PDF attachments, PDFs are not imported.

In an older organisation I _have_ seen PDFs that were once imported, and I'm inclined to think this was a glitch during older imports. I've checked the mime-type that WordPress used for those imported PDFs, and those seemed to be correctly set to `application/pdf`.

I'm keeping this issue open for now, perhaps I can reproduce it at another time.

**Comment by jkva** [comment]
This logic should be moved into the main `WordPressPlugin` class and generalised such that a `WP_Post` or `ID` can be provided and checked whether it should be processed. This relates to #90.
